WalletWasabi.Fluent/Global.cs

```diff
@@ -401,12 +405,15 @@ public async Task DisposeAsync()
 
 				try
 				{
-					await BitcoinStore.DisposeAsync().ConfigureAwait(false);
-					Logger.LogInfo($"{nameof(BitcoinStore)} is disposed.");
+					await IndexStore.DisposeAsync().ConfigureAwait(false);
+					Logger.LogInfo($"{nameof(IndexStore)} is disposed.");
+
+					await AllTransactionStore.DisposeAsync().ConfigureAwait(false);
+					Logger.LogInfo($"{nameof(AllTransactionStore)} is disposed.");
 				}
 				catch (Exception ex)
 				{
-					Logger.LogError($"Error during the disposal of {nameof(BitcoinStore)}: {ex}");
+					Logger.LogError($"Error during the disposal of {nameof(IndexStore)} and {nameof(AllTransactionStore)}: {ex}");
```

Comment:
The could also be a problem disposing `IndexStore`.

Reply:
I don't understand. That's what the sentence says, isn't it?

Reply:
What I mean is that the error is during disposal of `IndeStore` OR `AllTransactionStore`. The alternative could be to have a `ty-catch` or each of these. 
I don't know the `IDisposeAsync` pattern well enough but should the `DisposeAsync` be allowed to throw exception?

Reply:
I think the best practice (or what makes sense to me really) is not to throw exceptions disposing methods of `IDisposable` and `IAsyncDisposable`. 

Here, I basically tried to preserve what we had before. I would like to remove that `try-catch` actually as that would be imho ideal but I'm not sure why it was added so I did not remove it right away.